resources/public/api/conf/1.0/endpoints/lossClaims_create.raml

```diff
@@ -0,0 +1,42 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
```

Comment:
Check: scenario text should come from the scenario column of the spec not the message column.

Applies to FORMAT_NINO, FORMAT_TAX_YEAR, RULE_TYPE_OF_CLAIM_INVALID